### ISSUE TITLE
24.0.9 FVT: The Gradle dev command is taking longer than expected on the initial run.

### ISSUE THREAD

**Issue by aparnamichael** [OpenLiberty/ci.gradle#935]
When we run the start command for the first time in a Gradle project, it takes longer than expected and gives the impression that the dev mode is stuck. In the terminal, I can see the below message after dev mode starts.

`BUILD SUCCESSFUL in 4m 35s`

**Comment by cherylking** [comment]
> @TrevCraw Maybe I'm missing something but hasn't he identified that gradle is downloading dependencies and installing features? Things we just have to wait for?

Exactly, we cannot do anything about how long it takes to download dependencies. If this is the root cause, it means a subsequent start of dev mode should take much less time. Does it?

**Comment by gilbysunil14** [comment]
> > [@TrevCraw](https://github.com/TrevCraw) Maybe I'm missing something but hasn't he identified that gradle is downloading dependencies and installing features? Things we just have to wait for?
> 
> Exactly, we cannot do anything about how long it takes to download dependencies. If this is the root cause, it means a subsequent start of dev mode should take much less time. Does it?

Yes. The subsequent start of dev mode is taking much less time.
